src/ProjectAudioManager.cpp

```diff
@@ -937,6 +937,9 @@ bool ProjectAudioManager::DoRecord(AudacityProject &project,
          gAudioIO->AILAInitialize();
       #endif
 
+      if (gAudioIO->IsMonitoring())
+         gAudioIO->StopMonitoring();
+
```

Comment:
Is there a similar problem still to fix if you start play while monitoring?

Reply:
Good point! it happens also when starting to play, both in the latest master and on this branch at this point, so I will have to fix also that case. I am updating the issue description to include this case.